Rework alphalab phones (#629)
pupil-labs/pupil-docs: src/alpha-lab/phone-neon.md

<TagLinks />
<Youtube src="gp5O1uskDME"/>

::: tip
What catches your attention and how do you visually navigate through the interface/page of a mobile app? This guide shows you the tools you can use to address these questions. 
:::

## Gaze behaviour during mobile app engagement

> Researchers in fields such as UI/UX and neuromarketing frequently investigate user visual engagement with mobile applications. Wearable eye tracking is sometimes used for this purpose as it can provide valuable insights into user experience during mobile browsing. In this article, we tested the effectiveness of our latest wearable eye tracker, Neon, in capturing and analysing users' viewing behaviour when focusing on small icons and features of mobile applications.

Author: Reading through this again, I suggest we slightly re-frame the intent of the article. Rather than 'testing' the effectiveness of Neon for mobile browsing, we should say that we 'used' neon to investigate mobile browsing. I reason that it wasn't really a proper test. But by demonstrating we used it to examine phone browsing, it shows that it's possible and motivates people to try it, rather than making concrete claims.

> Researchers in fields such as UI/UX and neuromarketing frequently investigate user visual engagement with mobile applications. Wearable eye tracking is sometimes used for this purpose as it can provide valuable insights into user experience during mobile browsing. In this article, we tested the effectiveness of our latest wearable eye tracker, Neon, in capturing and analysing users' viewing behaviour when focusing on small icons and features of mobile applications.

Author: ```suggestion
Researchers in fields such as UI/UX and neuromarketing frequently investigate user visual engagement with mobile applications. Wearable eye tracking is sometimes used for this purpose as it can provide valuable insights into user experience during mobile browsing. In this article, we used our latest wearable eye tracker, Neon, to capture and analyse users' viewing behaviour when focusing on small icons and features of mobile applications.
```

## Overcoming usability challenges with Neon

> In the past, wearable eye tracking faced challenges in usability primarily due to calibration limitations, which may have made mobile browsing measurements difficult and time-consuming in practice. However, we believe that Neon overcomes these obstacles by offering calibration-free gaze estimation with very good accuracy. For this reason, we wanted to test Neon’s performance for tracking gaze during mobile browsing.

Author: ```suggestion
In the past, wearable eye tracking faced challenges in usability primarily due to calibration limitations, which may have made mobile browsing measurements difficult and time-consuming in practice. However, we believe that Neon overcomes these obstacles by offering calibration-free gaze estimation with very good accuracy. For this reason, we wanted to try out Neon when tracking gaze during mobile browsing.
```

## Leveraging Pupil Cloud and Alpha Lab Tutorials to explore user gaze dynamics

> In order to assess how Neon characterises viewing behaviour on mobile phones, we first needed to make some eye tracking recordings whilst a user was engaging with a phone. We then leveraged Pupil Cloud and Alpha Lab tutorials to interpret the data and compute further metrics. 

Author: ```suggestion
In order to characterise viewing behaviour on mobile phones using Neon, we first needed to make some eye tracking recordings whilst a user was engaging with a phone. We then leveraged Pupil Cloud and Alpha Lab tutorials to interpret the data and compute further metrics. 
```

Firstly, we placed the phone in a static position on a desk, allowing us to interact with the app. 

> Then, we used Pupil Cloud's [Reference Image Mapper](/enrichments/reference-image-mapper/) to map gaze onto a 2D reference image of the environment that contained our test phone, like in the videos and snapshots you will see next. 

Author: ```suggestion
Then, we used Pupil Cloud's [Reference Image Mapper](/enrichments/reference-image-mapper/) to map gaze onto a 2D reference image of the environment that contained our test phone, like in the videos and snapshots. 
```

> Subsequently, we exported the data to analyse offline for in-depth exploration. For the offline analysis, we used existing Alpha Lab tutorials that allowed us to [generate scanpaths](/alpha-lab/scanpath-rim/), [define areas of interest (AOIs) and calculate gaze metrics](/alpha-lab/gaze-metrics-in-aois/), and [map gaze onto dynamic content](/alpha-lab/map-your-gaze-to-a-2d-screen/). 

Author: ```suggestion
Subsequently, we exported the data to analyse offline for a more in-depth exploration. For the offline analysis, we used existing Alpha Lab tutorials that allowed us to [generate scanpaths](/alpha-lab/scanpath-rim/), [define areas of interest (AOIs) and calculate gaze metrics](/alpha-lab/gaze-metrics-in-aois/), and [map gaze onto dynamic content](/alpha-lab/map-your-gaze-to-a-2d-screen/). 
```

The following section presents the results of our analyses, each accompanied by a brief description for clarity.

### Heatmaps and scanpaths

> Two valuable visualisations for understanding gaze behaviour on mobile screens are heatmaps and scanpaths (illustrated in the left and right panels below). 

Author: ```suggestion
Two valuable visualisations for understanding gaze behaviour on mobile screens are heatmaps and scanpaths (illustrated in the left and right panel). 
```

Heatmaps visually represent the app areas that receive the most attention, with warmer colours denoting higher fixation rates. These heatmaps can be easily generated within Pupil Cloud using the Reference Image Mapper.

> Meanwhile, scanpaths depict the sequence of eye movements and fixations during visual exploration. The circle size in the scanpath corresponds to fixation duration; larger circles indicate longer user focus on specific screen areas. For those interested in generating these visualisations, our [scanpath tutorial](/alpha-lab/scanpath-rim/) provides a step-by-step guide.

Author: Also suggest we add a small description of what our results show in this case. For example, 'You can see the largest fixation circle shows most attention was paid initially to xxx region of the app'

> <div class="mcontainer">

Author: Let's move these images such that they appear first, underneath the section header

Author: Follow-up question. I can't remember if we already discussed this. But the heatmap seems not to correspond to the scanpath visualisation. That is, there are no fixations shown on the hottest area of the heatmap. Is that not counterintuitive?

</div>

### Calculation of gaze metrics on AOIs
Examining eye tracking data provides valuable insights into user behaviour, but relying solely on visualisations like heatmaps and scanpaths might not always reveal the complete picture. To gain a more comprehensive understanding, we conducted a quantitative analysis of our data. This involved calculating gaze metrics such as [dwell time](/alpha-lab/gaze-metrics-in-aois/#dwell-time) and [time to first contact](/alpha-lab/gaze-metrics-in-aois/#time-to-first-contact). These metrics offer tangible and quantitative outcomes about the salience of each AOI. 

> Dwell time signifies the total duration of fixation on a specific AOI, serving as an indicator of attentional allocation. Longer dwell times suggest prolonged focus on an AOI. On the other hand, the time to first contact measures how quickly an AOI captures a user's attention, indicating its prominence.

Author: We could also add a short discussion of what our results show here. It can be brief, but would round the section off nicely

For a detailed explanation and visual representation of these metrics, you can refer to our [AOI tutorial](/alpha-lab/gaze-metrics-in-aois/), which provides step-by-step instructions and charts for your analysis.

> <div class="pb-4" style="display:flex;justify-content:center;">

Author: Let's move these images so they appear first, underneath this sections header

  </v-img>
</div>

### Map your gaze onto dynamic phone screen content

> Until now, the tools that we have used have been limited to mapping gaze onto static images. However, the dynamic realm of mobile phones makes it important to study gaze behaviour in a more natural context, particularly during scrolling. We, therefore, used the [dynamic screen mapping tutorial](/alpha-lab/map-your-gaze-to-a-2d-screen/) for this purpose. This approach will provide a comprehensive understanding of gaze behaviour in dynamic interfaces.

Author: ```suggestion
Until now, the tools that we have used have been limited to mapping gaze onto static images. However, the dynamic realm of mobile phones makes it important to study gaze behaviour in a more natural context, particularly during scrolling. We, therefore, used the [dynamic screen mapping tutorial](/alpha-lab/map-your-gaze-to-a-2d-screen/) for this purpose. This approach provided a more comprehensive understanding of gaze behaviour in dynamic interfaces.
```

> Upon reviewing the recording, Neon demonstrates good accuracy in capturing gaze behaviour, offering a comprehensive high-level overview of the wearer's focal points.

Author: ```suggestion
Upon reviewing the recording, Neon demonstrated good accuracy in capturing gaze behaviour, offering a useful high-level overview of the wearer's focal points.
```

> While visualisations are valuable, a very useful facet of this tool is its ability to generate a CSV file containing gaze data mapped onto the screen, represented in 2D x, y coordinates. This feature opens avenues for extensive customisation and in-depth analysis.

Author: ```suggestion
While visualisations are valuable, a very useful facet of this tool is its ability to generate a CSV file containing gaze data mapped onto the screen, represented in 2D x, y coordinates. This feature opens avenues for extensive customisation and more in-depth analysis.
```

> <Youtube src="RKrf3YQjzao"/>

Author: Again, I think this would be better right under the header
